src/Compiler/FSComp.txt

```diff
@@ -1527,6 +1527,7 @@ notAFunctionButMaybeDeclaration,"This value is not a function and cannot be appl
 #3261 reserved for ConstraintSolverNullnessWarningWithTypes
 #3261 reserved for ConstraintSolverNullnessWarningWithType
 #3261 reserved for ConstraintSolverNullnessWarning
+3262,tcPassingWithoutNullToANullableExpectingFunc,"Value known to be without null passed to a function meant for nullables: %s"
```

Comment:
Wondering if it's good wording. Sounds a bit confusing. But we can review all messaging in the resulting PR.

Reply:
The second part of the naming comes from the attribute itself, which is (tries to be, suggestions welcome!) tailored to the situation.
See more in the tests, where the assertion contains the full text incl. the custom placeholder.

https://github.com/dotnet/fsharp/pull/16853/files#diff-b9def937a045fb1bfb790380a509648589c8a4fdd7399fd2141b25e00132a624R247